src/geom/Transform.cc

```diff
@@ -105,16 +105,22 @@ typename FromEndpoint::Array Transform<FromEndpoint, ToEndpoint>::tranInverse(
 }
 
 template <typename FromEndpoint, typename ToEndpoint>
+Transform<ToEndpoint, FromEndpoint> Transform<FromEndpoint, ToEndpoint>::getInverse() const {
+    std::shared_ptr<ast::Mapping> inverse = _frameSet->getInverse();
```

Comment:
Sure...create a FrameSet and add some frames. Then look for them after constructing the Transform and retrieving the FrameSet from it.